### ISSUE TITLE
Update A11y Testing Documentation to Include WCAG 2.2 Details

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#37034]
# Description
In order to prepare for the release of WCAG 2.2 standards in June 2022, we need to update our existing a11y testing documentation to include the new set of rules that will be included in WCAG 2.2.

## Acceptance Criteria

- [ ] Prepare a document focused on WCAG 2.2 rules similar to this [one ](https://depo-platform-documentation.scrollhelp.site/developer-docs/WCAG-2.1-success-criteria-and-foundational-testing.1777893393.html)for WCAG 2.1.

**Comment by noahgelman** [comment]
New WCAG 2.2 Success Criteria:

2.4.11 Focus Appearance (Minimum)
2.4.12 Focus Appearance (Enhanced)
 - Both of the above relate to the focus state of buttons, links, and other focusable elements. Our styles probably need updating. With Design, could probably take 3 weeks.
2.4.13
 - Relates more to publications or print related content. We won't need to do anything for this
2.5.7 Dragging Movements
 - Draggable elements can be done with 1 finger. We should already be compliant on this. There is another standard that covers single finger gestures. We won't need to do anything for this.
2.5.8 Target Size (Minimum)
 - This relates to buttons or clickable elements having a minimum size or minimum spacing aroudn it. We'll need to review our design and potentially update CSS. With Design, could probably take 2 weeks.
3.2.6 Consistent Help
 - Pages that offer help links need to have it consistantly placed on each page. We'll need to review our design and potentially HTML. Time would depend on how many affected pages there are
3.2.7 Accessible Authentication
 - Authentication accessibility standards are noted. Would need to review our process and potentially update it. Could take 3 weeks
3.3.8 Redundant entry
 - Information user already submitted should be auto-populated. We'll need to review and potentially update certain forms. Will likely be an ongoing process so the length of time can't be estimated

**Comment by noahgelman** [comment]
https://vfs.atlassian.net/wiki/spaces/pilot/pages/2099872052/Copy+of+WCAG+2.2+Success+Criteria+and+Foundational+Testing

**Comment by pjhill** [comment]
This is looking pretty good. I left a few comments. Please review those comments and then let's publish this.

**Comment by pjhill** [comment]
@noahgelman Once the comments I left in this document have been resolved please mark each comment as resolved.

**Comment by pjhill** [comment]
This document is good enough to publish. We can iterate and refine it later if need be.
